### ISSUE TITLE
focus state needs more contrast for low vision users

### ISSUE THREAD

**Issue by rlskoeser** [Princeton-CDH/lenape-timetree#262]
accessibility review feedback from @itmaybejj:

> Focus state often has very little contrast -- black on brown.

suggestion:
> "Ideally pick something very bright and contrasty. It may also make sense to pick two colors, so something always shines through regardless of the foreground and background:
``` css
a:focus-visible, button:focus-visible {
  outline: 3px solid transparent;
  box-shadow: 0 0 0 2px cyan, 0 0 0 3px black;
}"
```
